content-src/components/Card/Card.jsx

```diff
@@ -240,7 +239,7 @@ export class _Card extends React.PureComponent {
             <div className="card-context">
               {icon && !link.context && <span aria-haspopup="true" className={`card-context-icon icon icon-${icon}`} />}
               {link.icon && link.context && <span aria-haspopup="true" className="card-context-icon icon" style={{backgroundImage: `url('${link.icon}')`}} />}
-              {intlID && !link.context && <div className="card-context-label"><FormattedMessage id={intlID} defaultMessage="Visited" /></div>}
+              {fluentID && !link.context && <div className="card-context-label"><span data-l10n-id={fluentID} defaultMessage="Visited" /></div>}
```

Comment:
@mardak With fluent this renders as <span data-l10n-id="newtab-label-bookmarked" defaultmessage="Visited">Bookmarked</span>, is that ok?

Reply:
If you're referring to `defaultMessage="Visited"` I don't think we need to keep that? It shouldn't do anything anyway on the `span`.. so maybe the `data-l10n-id` should just be on the div ?